air/src/stack/op_flags/mod.rs

```diff
@@ -0,0 +1,642 @@
+use super::EvaluationFrame;
+use crate::utils::binary_not;
+use vm_core::{
+    decoder::{NUM_OP_BITS, OP_BITS_RANGE, OP_BIT_EXTRA_COL_IDX},
+    Felt, FieldElement, DECODER_TRACE_OFFSET, TRACE_WIDTH, ZERO,
+};
+
+#[cfg(test)]
+pub mod tests;
+
+// CONSTANTS
+// ==================================================================================================
+
+pub const NUM_DEGREE_7_OPS: usize = 64;
+pub const NUM_DEGREE_6_OPS: usize = 16;
+pub const NUM_DEGREE_4_OPS: usize = 8;
+
+pub struct OpFlags<E: FieldElement> {
```

Comment:
Let's add a comment here (a few sentences) describing what this struct is.

Reply:
done